classes/notice_origin.py

```diff
@@ -50,10 +50,10 @@ def print_notices(self):
                 warnings = warnings + notice.message
             if notice.level == 'error':
                 errors = errors + notice.message
-            if notice.level == 'hints':
+            if notice.level == 'hint':
                 hints = hints + notice.message
-        notice_msg = formats.notice_format(
-            origin=self.origin, info=info, warnings=warnings, errors=errors,
+        notice_msg = formats.notice_format.format(
+            origin=self.__origin_str, info=info, warnings=warnings, errors=errors,
```

Comment:
Could you change this to `self.origin_str`? There is a property getter defined for it. Thanks!